packages/web/components/ui/media-file-node.tsx

```
@@ -19,31 +23,31 @@ export const FileElement = withHOC(
+      <PlateElement className='my-px rounded-sm' {...props}>
         <a
-          className="group relative m-0 flex cursor-pointer items-center rounded px-0.5 py-[3px] hover:bg-muted"
+          className='group relative m-0 flex cursor-pointer items-center rounded px-0.5 py-[3px] hover:bg-muted'
           contentEditable={false}
           download={name}
           href={unsafeUrl}
-          rel="noopener noreferrer"
-          role="button"
-          target="_blank"
+          rel='noopener noreferrer'
+          role='button'
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)